src/Apps/W1/Subscription Billing/App/Billing/Pages/ContractBillingErrLog.Page.al

```diff
@@ -0,0 +1,71 @@
+namespace Microsoft.SubscriptionBilling;
+
+page 8113 "Contract Billing Err Log"
+{
+    Caption = 'Contract Billing Error Log';
+    PageType = List;
+    ApplicationArea = All;
+    SourceTable = "Contract Billing Err Log";
```

Comment:
I couldn't find the process that deletes errors in this log.
What's the process to delete thos error logs?

Reply:
Valid concern. The thing is that the 2nd run should skip the processing of previously problematic entries as we do not want to flood the error log - the Job Queue runs every hour. The current idea is that those Billing Lines that run on error can be processed manually. Billing Line contains reference to an Error Log Entry which in case of Log deletion should be cleared. The area of error handling was not that well covered with consulting concept I received from consulting, so I am open to suggestions.